src/index.html

```diff
@@ -7,16 +7,16 @@
   <title>Document</title>
 </head>
 <body>
-  <div class="to-do-list-div">
+  <div class="container">
     <div class="title">
       <p>Today's To Do</p>
     </div>
-    <ul id="to-do-list" class="to-do-list">
-      <li class="list-item">
-        <input type="text" placeholder="Add to your list....">
-      </li>
-    </ul>
-    <button>Clear all completed</button>
+    <div class="to-do-task">
```

Comment:
- [ Required ] Please consider following the design provided for this project, currently, your design does not match the design provided. Please see the highlighted sections in the screenshots below for more details on this.

## Project design
<img src="https://user-images.githubusercontent.com/16392046/221179112-159fbce3-c98d-4c0f-a3eb-3d9a3a609f2f.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/221179343-99246bdd-dfd2-4cfe-9220-1e9a808b2cf8.png" width="375px" />